examples/oauth2/oauth2_chrome_test.py

```diff
@@ -16,7 +16,8 @@ def test_chrome():
         'oauth2_client_secret': os.environ['OAUTH2_CLIENT_SECRET'],
         'oauth2_authz_endpoint_url': os.environ['OAUTH2_AUTHZ_ENDPOINT_URL'],
         'oauth2_token_endpoint_url': os.environ['OAUTH2_TOKEN_ENDPOINT_URL'],
-        'scopes': os.environ['OAUTH2_SCOPES'],
+        'oauth2_redirect_uri': os.getenv('OAUTH2_REDIRECT_URI', ''),
```

Comment:
`redirect_uri` is only required if a client has multiple redirect_uris.
https://tools.ietf.org/html/rfc6749#section-4.1.1